platform/tests/TESTS/mbed_platform/minimal-printf/compliance/main.cpp

```diff
@@ -914,6 +1061,14 @@ static control_t test_printf_f(const size_t call_count)
     result_minimal = mbed_printf("f: %f\r\n", 2.12345649);
     result_baseline = sprintf(buffer_baseline, "f: 2.123456\r\n");
     TEST_ASSERT_EQUAL_INT(result_baseline, result_minimal);
+    CLEAN_BUFFER;
+    result_minimal = mbed_printf("f: %.3f\r\n", 2.12345649);
+    result_baseline = sprintf(buffer_baseline, "f: 2.123\r\n");
+    TEST_ASSERT_EQUAL_INT(result_baseline, result_minimal);
+    CLEAN_BUFFER;
+    result_minimal = mbed_printf("f: %.8f\r\n", 2.12345649);
+    result_baseline = sprintf(buffer_baseline, "f: 2.12345649\r\n");
+    TEST_ASSERT_EQUAL_INT(result_baseline, result_minimal);
```

Comment:
Add some tests for `width_size` and prepending character.

Reply:
Done